packages/core/src/utils.ts

```diff
@@ -51,7 +50,8 @@ export function convertToTimestampToDateTime(timestamp: number): Date {
 export function isJson(data: any) {
   return (
     (isArray(data) || isObject(data)) &&
-    !isBuffer(data) &&
+    !Buffer.isBuffer(data) &&
```

Comment:
Passes the test (runs in node) but crashes in a browser because there is no global `Buffer` in browsers.